src/scss/generic/_body.scss

```diff
@@ -1,9 +1,12 @@
 body {
+  -webkit-font-smoothing: antialiased;
```

Comment:
@eduardojmatos it's already there, I don't know why is not showing the change. Take a look a the file. 👍